sherpa/astro/io/tests/test_io_pha.py

```diff
@@ -1276,10 +1276,16 @@ def test_write_pha_missing_column(column, is_ascii, tmp_path):
     At the moment we write out the file.
     """
 
-    chans = np.arange(1, 4, dtype=np.int32)
-    counts = np.arange(1, 4, dtype=np.int32)
+    if column == "channel":
```

Comment:
I' confused by "if column == channel, then chans = None" instead of the other way around, but I guess in te context of this test that makes sense.